### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by timvisee**
Some minor nitpicks or improvements of things I came across that didn't really fit in another PR.

See commits for specific changes.

I'll keep doing these if they appear useful.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### CODE REVIEW

File: src/common/helpers.rs [hunk 1]
```diff
@@ -39,7 +39,6 @@ pub fn create_search_runtime(max_search_threads: usize) -> io::Result<Runtime> {
 
     runtime::Builder::new_multi_thread()
         .worker_threads(search_threads)
-        .enable_time()
```
Comment (generall) [inline src/common/helpers.rs:42]: Do you know if there is any Drawback of using `enable_time`?

**Review by generall — APPROVED** [review]

File: src/common/helpers.rs (hunk 1 above)
Comment (timvisee) [reply src/common/helpers.rs:42]: In this case, no drawbacks other than it being obsolete, as it is also enabled with `enable_all`.

Or do you mean what the drawback may be of it enabling at all?

Comment (generall) [reply src/common/helpers.rs:42]: I mean, is there an overhead for this option or something? Why don't we just enable it everywhere

Comment (timvisee) [reply src/common/helpers.rs:42]: Since this doesn't affect this PR itself I'll merge now.

**Pull request merged**